### PR TITLE
Fixed line breaking on right side

### PR DESCRIPTION
**Pull request by AtlassianRS**
This should correct the calculation when multiple lines appear on the right side. 
Has been tested in some scenarios.

### CODE REVIEW

**Comment by Nightfirecat** [comment]
The code looks fine at a glance, will test this further tomorrow.

**Comment by AtlassianRS** [comment]
Found a bug in the original pull request. If the left text was too long, it would compute a negative size for the right maxwidth. I've reverted the calculation of the rightSmallWidth to preferredsize / 3. It looks like the only calculation issue was in the position of the right text.

**Comment by Nightfirecat** [comment]
fixed

:tada:

File: runelite-client/src/main/java/net/runelite/client/ui/overlay/components/LineComponent.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public Dimension render(Graphics2D graphics)
 
 			if (!Strings.isNullOrEmpty(right))
 			{
-				rightSmallWidth = (preferredSize.width / 3);
+				rightSmallWidth = preferredSize.width / 3;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/ui/overlay/components/LineComponent.java]: If there's no functional change, can the changes be reverted on this line?

File: runelite-client/src/main/java/net/runelite/client/ui/overlay/components/LineComponent.java [hunk 2]
```diff
@@ -170,5 +170,4 @@ private static int getLineWidth(final String line, final FontMetrics metrics)
 
 		return wrapped.toString().split("\n");
 	}
-}
-
+}
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/ui/overlay/components/LineComponent.java]: Revert the newline at EOF change.

**Pull request merged**